dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportController.java

```diff
@@ -160,7 +161,8 @@ public TrackedEntitiesExportController(
   ResponseEntity<Page<ObjectNode>> getTrackedEntities(TrackedEntityRequestParams requestParams)
       throws BadRequestException, ForbiddenException, NotFoundException {
     validatePaginationParameters(requestParams);
-    TrackedEntityOperationParams operationParams = paramsMapper.map(requestParams);
+    TrackedEntityOperationParams operationParams =
+        paramsMapper.map(requestParams, CurrentUserUtil.getCurrentUserDetails());
```

Comment:
I see `@CurrentUser UserDetails currentUser` being used in controllers. Not sure if we should stick to one of the ways